[2025-03-25 12:11:52] [31merror[39m: This is an error log
[2025-03-25 12:16:39] [31merror[39m: This is an error log
[2025-04-08 16:06:04] [31merror[39m: Service ID Not Found: 
[2025-04-08 16:39:17] [31merror[39m: Service ID Not Found: 
[2025-04-08 16:39:26] [31merror[39m: Auth Create User Service:  
Invalid `prisma.user.create()` invocation in
D:\Programing\WorkFolders\restaurant-pos\backend\src\domain\auth\auth.repository.ts:5:22

  2 import { User } from "../../types/common";
  3 
  4 export async function createUserDB(username: string, password: string) {
â†’ 5   return prisma.user.create(
Can't reach database server at `yamanote.proxy.rlwy.net:42595`

Please make sure your database server is running at `yamanote.proxy.rlwy.net:42595`.
[2025-04-08 16:47:53] [31merror[39m: Auth Create User Service:  
Invalid `prisma.user.create()` invocation in
D:\Programing\WorkFolders\restaurant-pos\backend\src\domain\auth\auth.repository.ts:5:22

  2 import { User } from "../../types/common";
  3 
  4 export async function createUserDB(username: string, password: string) {
â†’ 5   return prisma.user.create(
The column `username` does not exist in the current database.
[2025-04-08 16:49:54] [31merror[39m: Auth Create User Service:  
Invalid `prisma.user.create()` invocation in
D:\Programing\WorkFolders\restaurant-pos\backend\src\domain\auth\auth.repository.ts:5:22

  2 import { User } from "../../types/common";
  3 
  4 export async function createUserDB(username: string, password: string) {
â†’ 5   return prisma.user.create(
The column `username` does not exist in the current database.
[2025-04-08 16:51:27] [31merror[39m: Auth Create User Service:  
Invalid `prisma.user.create()` invocation in
D:\Programing\WorkFolders\restaurant-pos\backend\src\domain\auth\auth.repository.ts:5:22

  2 import { User } from "../../types/common";
  3 
  4 export async function createUserDB(username: string, password: string) {
â†’ 5   return prisma.user.create(
The column `username` does not exist in the current database.
[2025-04-08 16:56:23] [31merror[39m: Service ID Not Found: 
[2025-04-08 16:56:29] [31merror[39m: Service ID Not Found: 
[2025-04-08 16:56:31] [31merror[39m: Service ID Not Found: 
[2025-04-08 16:56:40] [31merror[39m: Service ID Not Found: 
[2025-04-08 16:56:41] [31merror[39m: Service ID Not Found: 
[2025-04-08 16:56:41] [31merror[39m: Service ID Not Found: 
[2025-04-08 16:56:42] [31merror[39m: Service ID Not Found: 
[2025-04-08 16:58:58] [31merror[39m: Service ID Not Found: 
[2025-04-08 17:09:14] [31merror[39m: Validation failed: [
  {
    "code": "invalid_type",
    "expected": "number",
    "received": "string",
    "path": [
      "id"
    ],
    "message": "Expected number, received string"
  }
]
[2025-04-08 17:09:14] [31merror[39m: Service ID Not Found: 
[2025-04-08 17:10:20] [31merror[39m: Validation failed: [
  {
    "code": "invalid_type",
    "expected": "number",
    "received": "string",
    "path": [
      "id"
    ],
    "message": "Expected number, received string"
  }
]
[2025-04-08 17:10:20] [31merror[39m: Service ID Not Found: 
[2025-04-08 17:11:58] [31merror[39m: Validation failed: [
  {
    "code": "invalid_type",
    "expected": "number",
    "received": "string",
    "path": [
      "id"
    ],
    "message": "Expected number, received string"
  }
]
[2025-04-08 17:11:58] [31merror[39m: Service ID Not Found: 
[2025-04-08 17:13:17] [31merror[39m: Service User Not Found: 
[2025-04-08 17:13:57] [31merror[39m: Validation failed: [
  {
    "code": "invalid_type",
    "expected": "number",
    "received": "string",
    "path": [
      "id"
    ],
    "message": "Expected number, received string"
  }
]
[2025-04-08 17:14:30] [31merror[39m: Service User Not Found: 
[2025-04-08 17:29:59] [31merror[39m: Validation failed: [
  {
    "code": "invalid_type",
    "expected": "string",
    "received": "undefined",
    "path": [
      "password"
    ],
    "message": "Required"
  },
  {
    "expected": "'ADMIN' | 'USER'",
    "received": "undefined",
    "code": "invalid_type",
    "path": [
      "role"
    ],
    "message": "Required"
  }
]
[2025-04-08 17:30:56] [31merror[39m: Auth Update User Service:  
Invalid `prisma.user.update()` invocation in
D:\Programing\WorkFolders\restaurant-pos\backend\src\domain\auth\auth.repository.ts:35:22

  32 }
  33 
  34 export async function updateUserDB(id: number, data: User) {
â†’ 35   return prisma.user.update(
Can't reach database server at `yamanote.proxy.rlwy.net:42595`

Please make sure your database server is running at `yamanote.proxy.rlwy.net:42595`.
[2025-04-08 17:36:37] [31merror[39m: Service User Not Found: 
[2025-04-08 17:36:45] [31merror[39m: Auth Delete User Service:  
Invalid `prisma.user.delete()` invocation in
D:\Programing\WorkFolders\restaurant-pos\backend\src\domain\auth\auth.repository.ts:11:28

   8 }
   9 
  10 export async function deleteUserDB(id: number) {
â†’ 11   return prisma.user.delete(
An operation failed because it depends on one or more records that were required but not found. Record to delete does not exist.
[2025-04-09 14:41:10] [31merror[39m: Auth login Service:  
Invalid `prisma.user.findFirst()` invocation in
D:\Programing\WorkFolders\restaurant-pos\backend\src\domain\auth\auth.repository.ts:27:22

  24 }
  25 
  26 export async function findUserNameDB(username: string) {
â†’ 27   return prisma.user.findFirst(
Can't reach database server at `yamanote.proxy.rlwy.net:42595`

Please make sure your database server is running at `yamanote.proxy.rlwy.net:42595`.
[2025-04-09 15:35:40] [31merror[39m: Auth login Service:  
Invalid `prisma.user.findFirst()` invocation in
D:\Programing\WorkFolders\restaurant-pos\backend\src\domain\auth\auth.repository.ts:27:22

  24 }
  25 
  26 export async function findUserNameDB(username: string) {
â†’ 27   return prisma.user.findFirst(
Can't reach database server at `yamanote.proxy.rlwy.net:42595`

Please make sure your database server is running at `yamanote.proxy.rlwy.net:42595`.
[2025-04-09 15:38:00] [31merror[39m: Authenticated middleware:  next is not a function
[2025-04-09 15:43:29] [31merror[39m: Authenticated middleware:  next is not a function
[2025-04-09 15:48:31] [31merror[39m: Authenticated middleware:  next is not a function
[2025-04-09 15:56:18] [31merror[39m: Authenticated middleware:  
Invalid `prisma.user.findFirst()` invocation in
D:\Programing\WorkFolders\restaurant-pos\backend\src\domain\auth\auth.repository.ts:19:22

  16 }
  17 
  18 export async function findUserDB(id: number) {
â†’ 19   return prisma.user.findFirst(
Can't reach database server at `yamanote.proxy.rlwy.net:42595`

Please make sure your database server is running at `yamanote.proxy.rlwy.net:42595`.
[2025-04-09 15:56:33] [31merror[39m: Authenticated middleware:  next is not a function
[2025-04-09 16:16:14] [31merror[39m: Auth login Service:  
Invalid `prisma.user.findFirst()` invocation in
D:\Programing\WorkFolders\restaurant-pos\backend\src\domain\auth\auth.repository.ts:27:22

  24 }
  25 
  26 export async function findUserNameDB(username: string) {
â†’ 27   return prisma.user.findFirst(
Can't reach database server at `yamanote.proxy.rlwy.net:42595`

Please make sure your database server is running at `yamanote.proxy.rlwy.net:42595`.
[2025-04-09 16:17:00] [31merror[39m: Authenticated middleware:  next is not a function
[2025-04-09 16:17:02] [31merror[39m: Authenticated middleware:  next is not a function
[2025-04-09 16:19:04] [31merror[39m: Authenticated middleware:  next is not a function
[2025-04-09 16:26:09] [31merror[39m: Authenticated middleware:  next is not a function
[2025-04-09 16:26:11] [31merror[39m: Authenticated middleware:  next is not a function
[2025-04-09 16:28:00] [31merror[39m: Authenticated middleware:  next is not a function
[2025-04-09 16:28:48] [31merror[39m: Authenticated middleware:  next is not a function
[2025-04-09 16:37:53] [31merror[39m: Authenticated middleware:  
Invalid `prisma.user.findFirst()` invocation in
D:\Programing\WorkFolders\restaurant-pos\backend\src\domain\auth\auth.repository.ts:19:22

  16 }
  17 
  18 export async function findUserDB(id: number) {
â†’ 19   return prisma.user.findFirst(
Can't reach database server at `yamanote.proxy.rlwy.net:42595`

Please make sure your database server is running at `yamanote.proxy.rlwy.net:42595`.
[2025-04-09 17:03:42] [31merror[39m: Role: 
[2025-04-09 17:04:04] [31merror[39m: Admin Middleware:  colors[Colorizer.allColors[lookup]] is not a function
[2025-04-09 17:04:07] [31merror[39m: Admin Middleware:  colors[Colorizer.allColors[lookup]] is not a function
[2025-04-09 17:06:01] [31merror[39m: User: 
[2025-04-09 17:20:12] [31merror[39m: Auth login Service:  
Invalid `prisma.user.findFirst()` invocation in
D:\Programing\WorkFolders\restaurant-pos\backend\src\domain\auth\auth.repository.ts:27:22

  24 }
  25 
  26 export async function findUserNameDB(username: string) {
â†’ 27   return prisma.user.findFirst(
Can't reach database server at `yamanote.proxy.rlwy.net:42595`

Please make sure your database server is running at `yamanote.proxy.rlwy.net:42595`.
[2025-04-09 17:42:31] [31merror[39m: Authenticated middleware:  Cannot read properties of undefined (reading 'session')
[2025-04-09 17:42:33] [31merror[39m: Authenticated middleware:  Cannot read properties of undefined (reading 'session')
[2025-04-09 17:44:26] [31merror[39m: Auth login Service:  
Invalid `prisma.user.findFirst()` invocation in
D:\Programing\WorkFolders\restaurant-pos\backend\src\domain\auth\auth.repository.ts:27:22

  24 }
  25 
  26 export async function findUserNameDB(username: string) {
â†’ 27   return prisma.user.findFirst(
Can't reach database server at `yamanote.proxy.rlwy.net:42595`

Please make sure your database server is running at `yamanote.proxy.rlwy.net:42595`.
[2025-04-09 18:06:09] [31merror[39m: Auth login Service:  
Invalid `prisma.user.findFirst()` invocation in
D:\Programing\WorkFolders\restaurant-pos\backend\src\domain\auth\auth.repository.ts:27:22

  24 }
  25 
  26 export async function findUserNameDB(username: string) {
â†’ 27   return prisma.user.findFirst(
Can't reach database server at `yamanote.proxy.rlwy.net:42595`

Please make sure your database server is running at `yamanote.proxy.rlwy.net:42595`.
[2025-04-09 18:21:26] [31merror[39m: Authenticated middleware:  Cannot read properties of undefined (reading 'session')
[2025-04-09 18:21:28] [31merror[39m: Authenticated middleware:  Cannot read properties of undefined (reading 'session')
[2025-04-09 18:21:30] [31merror[39m: Authenticated middleware:  Cannot read properties of undefined (reading 'session')
[2025-04-09 18:21:31] [31merror[39m: Authenticated middleware:  Cannot read properties of undefined (reading 'session')
[2025-04-09 18:21:31] [31merror[39m: Authenticated middleware:  Cannot read properties of undefined (reading 'session')
[2025-04-09 18:21:32] [31merror[39m: Authenticated middleware:  Cannot read properties of undefined (reading 'session')
[2025-04-09 18:21:38] [31merror[39m: Authenticated middleware:  Cannot read properties of undefined (reading 'session')
[2025-04-09 18:27:09] [31merror[39m: Authenticated middleware:  Cannot destructure property 'session' of '(intermediate value)' as it is undefined.
[2025-04-09 18:27:11] [31merror[39m: Authenticated middleware:  Cannot destructure property 'session' of '(intermediate value)' as it is undefined.
[2025-04-09 18:31:35] [31merror[39m: Authenticated middleware:  Cannot destructure property 'email' of '(0 , decodeJWT_1.decodeJWT)(...)' as it is null.
[2025-04-09 18:31:36] [31merror[39m: Authenticated middleware:  Cannot destructure property 'email' of '(0 , decodeJWT_1.decodeJWT)(...)' as it is null.
[2025-04-09 18:31:37] [31merror[39m: Authenticated middleware:  Cannot destructure property 'email' of '(0 , decodeJWT_1.decodeJWT)(...)' as it is null.
[2025-04-09 18:43:41] [31merror[39m: Auth login Service:  
Invalid `prisma.user.findFirst()` invocation in
D:\Programing\WorkFolders\restaurant-pos\backend\src\domain\auth\auth.repository.ts:27:22

  24 }
  25 
  26 export async function findUserNameDB(username: string) {
â†’ 27   return prisma.user.findFirst(
Can't reach database server at `yamanote.proxy.rlwy.net:42595`

Please make sure your database server is running at `yamanote.proxy.rlwy.net:42595`.
[2025-04-09 18:43:53] [31merror[39m: Authenticated middleware:  Cannot destructure property 'email' of '(0 , decodeJWT_1.decodeJWT)(...)' as it is null.
[2025-04-09 18:43:55] [31merror[39m: Authenticated middleware:  Cannot destructure property 'email' of '(0 , decodeJWT_1.decodeJWT)(...)' as it is null.
[2025-04-09 18:52:44] [31merror[39m: Authenticated middleware:  Cannot destructure property 'username' of '(0 , decodeJWT_1.decodeJWT)(...)' as it is null.
[2025-04-09 18:53:51] [31merror[39m: Auth login Service:  
Invalid `prisma.user.findFirst()` invocation in
D:\Programing\WorkFolders\restaurant-pos\backend\src\domain\auth\auth.repository.ts:27:22

  24 }
  25 
  26 export async function findUserNameDB(username: string) {
â†’ 27   return prisma.user.findFirst(
Can't reach database server at `yamanote.proxy.rlwy.net:42595`

Please make sure your database server is running at `yamanote.proxy.rlwy.net:42595`.
[2025-04-09 18:56:36] [31merror[39m: Authenticated middleware:  colors[Colorizer.allColors[lookup]] is not a function
[2025-04-09 19:00:48] [31merror[39m: Auth login Service:  
Invalid `prisma.user.findFirst()` invocation in
D:\Programing\WorkFolders\restaurant-pos\backend\src\domain\auth\auth.repository.ts:27:22

  24 }
  25 
  26 export async function findUserNameDB(username: string) {
â†’ 27   return prisma.user.findFirst(
Can't reach database server at `yamanote.proxy.rlwy.net:42595`

Please make sure your database server is running at `yamanote.proxy.rlwy.net:42595`.
[2025-04-09 19:00:54] [31merror[39m: Auth login Service:  
Invalid `prisma.user.findFirst()` invocation in
D:\Programing\WorkFolders\restaurant-pos\backend\src\domain\auth\auth.repository.ts:27:22

  24 }
  25 
  26 export async function findUserNameDB(username: string) {
â†’ 27   return prisma.user.findFirst(
Can't reach database server at `yamanote.proxy.rlwy.net:42595`

Please make sure your database server is running at `yamanote.proxy.rlwy.net:42595`.
[2025-04-09 19:01:02] [31merror[39m: Auth login Service:  
Invalid `prisma.user.findFirst()` invocation in
D:\Programing\WorkFolders\restaurant-pos\backend\src\domain\auth\auth.repository.ts:27:22

  24 }
  25 
  26 export async function findUserNameDB(username: string) {
â†’ 27   return prisma.user.findFirst(
Can't reach database server at `yamanote.proxy.rlwy.net:42595`

Please make sure your database server is running at `yamanote.proxy.rlwy.net:42595`.
[2025-04-09 19:51:19] [31merror[39m: Authenticated middleware:  
Invalid `prisma.user.findFirst()` invocation in
D:\Programing\WorkFolders\restaurant-pos\backend\src\domain\auth\auth.repository.ts:19:22

  16 }
  17 
  18 export async function findUserDB(id: number) {
â†’ 19   return prisma.user.findFirst(
Can't reach database server at `yamanote.proxy.rlwy.net:42595`

Please make sure your database server is running at `yamanote.proxy.rlwy.net:42595`.
[2025-04-12 15:07:31] [31merror[39m: Auth login Service:  
Invalid `prisma.user.findFirst()` invocation in
D:\Programing\WorkFolders\restaurant-pos\backend\src\domain\auth\auth.repository.ts:27:22

  24 }
  25 
  26 export async function findUserNameDB(username: string) {
â†’ 27   return prisma.user.findFirst(
Can't reach database server at `yamanote.proxy.rlwy.net:42595`

Please make sure your database server is running at `yamanote.proxy.rlwy.net:42595`.
[2025-04-12 15:07:39] [31merror[39m: Auth login Service:  
Invalid `prisma.user.findFirst()` invocation in
D:\Programing\WorkFolders\restaurant-pos\backend\src\domain\auth\auth.repository.ts:27:22

  24 }
  25 
  26 export async function findUserNameDB(username: string) {
â†’ 27   return prisma.user.findFirst(
Can't reach database server at `yamanote.proxy.rlwy.net:42595`

Please make sure your database server is running at `yamanote.proxy.rlwy.net:42595`.
[2025-04-13 13:50:15] [31merror[39m: Authenticated middleware:  
Invalid `prisma.user.findFirst()` invocation in
D:\Programing\WorkFolders\restaurant-pos\backend\src\domain\auth\auth.repository.ts:19:22

  16 }
  17 
  18 export async function findUserDB(id: number) {
â†’ 19   return prisma.user.findFirst(
Can't reach database server at `yamanote.proxy.rlwy.net:42595`

Please make sure your database server is running at `yamanote.proxy.rlwy.net:42595`.
[2025-04-13 13:50:22] [31merror[39m: Authenticated middleware:  
Invalid `prisma.user.findFirst()` invocation in
D:\Programing\WorkFolders\restaurant-pos\backend\src\domain\auth\auth.repository.ts:19:22

  16 }
  17 
  18 export async function findUserDB(id: number) {
â†’ 19   return prisma.user.findFirst(
The column `User.image` does not exist in the current database.
[2025-04-13 15:54:59] [31merror[39m: Authenticated middleware:  Cannot destructure property 'username' of '(0 , decodeJWT_1.decodeJWT)(...)' as it is null.
[2025-04-13 15:55:01] [31merror[39m: Authenticated middleware:  Cannot destructure property 'username' of '(0 , decodeJWT_1.decodeJWT)(...)' as it is null.
[2025-04-13 15:55:01] [31merror[39m: Authenticated middleware:  Cannot destructure property 'username' of '(0 , decodeJWT_1.decodeJWT)(...)' as it is null.
[2025-04-13 15:55:02] [31merror[39m: Authenticated middleware:  Cannot destructure property 'username' of '(0 , decodeJWT_1.decodeJWT)(...)' as it is null.
[2025-04-13 16:20:54] [31merror[39m: Validation failed: [
  {
    "code": "invalid_type",
    "expected": "object",
    "received": "string",
    "path": [],
    "message": "Expected object, received string"
  }
]
[2025-04-13 16:21:34] [31merror[39m: Validation failed: [
  {
    "code": "invalid_type",
    "expected": "object",
    "received": "string",
    "path": [],
    "message": "Expected object, received string"
  }
]
[2025-04-13 16:22:22] [31merror[39m: Validation failed: [
  {
    "code": "invalid_type",
    "expected": "object",
    "received": "string",
    "path": [],
    "message": "Expected object, received string"
  }
]
[2025-04-13 16:23:50] [31merror[39m: Validation failed: [
  {
    "code": "invalid_type",
    "expected": "object",
    "received": "string",
    "path": [],
    "message": "Expected object, received string"
  }
]
[2025-04-13 16:24:13] [31merror[39m: Validation failed: [
  {
    "code": "invalid_type",
    "expected": "object",
    "received": "string",
    "path": [],
    "message": "Expected object, received string"
  }
]
[2025-04-13 16:24:28] [31merror[39m: Validation failed: [
  {
    "code": "invalid_type",
    "expected": "object",
    "received": "string",
    "path": [],
    "message": "Expected object, received string"
  }
]
[2025-04-13 16:26:22] [31merror[39m: Validation failed: [
  {
    "code": "invalid_type",
    "expected": "object",
    "received": "string",
    "path": [],
    "message": "Expected object, received string"
  }
]
[2025-04-13 16:26:39] [31merror[39m: Validation failed: [
  {
    "code": "invalid_type",
    "expected": "object",
    "received": "string",
    "path": [],
    "message": "Expected object, received string"
  }
]
[2025-04-13 16:26:42] [31merror[39m: Validation failed: [
  {
    "code": "invalid_type",
    "expected": "object",
    "received": "string",
    "path": [],
    "message": "Expected object, received string"
  }
]
[2025-04-13 16:28:34] [31merror[39m: Validation failed: [
  {
    "code": "invalid_type",
    "expected": "object",
    "received": "number",
    "path": [],
    "message": "Expected object, received number"
  }
]
[2025-04-13 16:28:36] [31merror[39m: Validation failed: [
  {
    "code": "invalid_type",
    "expected": "object",
    "received": "number",
    "path": [],
    "message": "Expected object, received number"
  }
]
[2025-04-13 16:29:26] [31merror[39m: Validation failed: [
  {
    "code": "invalid_type",
    "expected": "object",
    "received": "number",
    "path": [],
    "message": "Expected object, received number"
  }
]
[2025-04-13 16:29:29] [31merror[39m: Validation failed: [
  {
    "code": "invalid_type",
    "expected": "object",
    "received": "number",
    "path": [],
    "message": "Expected object, received number"
  }
]
[2025-04-15 13:34:32] [31merror[39m: Menu item Service:  
Invalid `prisma.menuItem.create()` invocation in
D:\Programing\WorkFolders\restaurant-pos\backend\src\domain\item\item.repository.ts:17:26

  14 }
  15 
  16 export async function createItemDB(data: MenuItem) {
â†’ 17   return prisma.menuItem.create({
         data: {
           title_en: "Burger EN",
           title_ar: "Burger AR",
           description_en: "Description of the burger in english",
           description_ar: "Description of the burger in arabi",
           description_ku: "Description of the burger in kurdish",
           price: 19.99,
           isActive: true,
       +   title_ku: String
         }
       })

Argument `title_ku` is missing.
[2025-04-15 13:34:39] [31merror[39m: Menu item Service:  
Invalid `prisma.menuItem.create()` invocation in
D:\Programing\WorkFolders\restaurant-pos\backend\src\domain\item\item.repository.ts:17:26

  14 }
  15 
  16 export async function createItemDB(data: MenuItem) {
â†’ 17   return prisma.menuItem.create({
         data: {
           title_en: "Burger EN",
           title_ar: "Burger AR",
           description_en: "Description of the burger in english",
           description_ar: "Description of the burger in arabi",
           description_ku: "Description of the burger in kurdish",
           price: 19.99,
           isActive: true,
       +   title_ku: String
         }
       })

Argument `title_ku` is missing.
[2025-04-15 13:34:40] [31merror[39m: Menu item Service:  
Invalid `prisma.menuItem.create()` invocation in
D:\Programing\WorkFolders\restaurant-pos\backend\src\domain\item\item.repository.ts:17:26

  14 }
  15 
  16 export async function createItemDB(data: MenuItem) {
â†’ 17   return prisma.menuItem.create({
         data: {
           title_en: "Burger EN",
           title_ar: "Burger AR",
           description_en: "Description of the burger in english",
           description_ar: "Description of the burger in arabi",
           description_ku: "Description of the burger in kurdish",
           price: 19.99,
           isActive: true,
       +   title_ku: String
         }
       })

Argument `title_ku` is missing.
[2025-04-15 13:34:41] [31merror[39m: Menu item Service:  
Invalid `prisma.menuItem.create()` invocation in
D:\Programing\WorkFolders\restaurant-pos\backend\src\domain\item\item.repository.ts:17:26

  14 }
  15 
  16 export async function createItemDB(data: MenuItem) {
â†’ 17   return prisma.menuItem.create({
         data: {
           title_en: "Burger EN",
           title_ar: "Burger AR",
           description_en: "Description of the burger in english",
           description_ar: "Description of the burger in arabi",
           description_ku: "Description of the burger in kurdish",
           price: 19.99,
           isActive: true,
       +   title_ku: String
         }
       })

Argument `title_ku` is missing.
[2025-04-15 13:44:36] [31merror[39m: Create Subcategory Service:  Cannot read properties of undefined (reading 'map')
[2025-04-15 13:45:46] [31merror[39m: Create Category Service:  Cannot read properties of undefined (reading 'map')
[2025-04-15 13:46:54] [31merror[39m: Validation failed: [
  {
    "code": "invalid_type",
    "expected": "array",
    "received": "number",
    "path": [
      "subCategory"
    ],
    "message": "Expected array, received number"
  }
]
[2025-04-15 13:47:07] [31merror[39m: Validation failed: [
  {
    "code": "invalid_type",
    "expected": "array",
    "received": "string",
    "path": [
      "subCategory"
    ],
    "message": "Expected array, received string"
  }
]
[2025-04-15 13:47:32] [31merror[39m: Create Category Service:  Cannot read properties of undefined (reading 'map')
[2025-04-15 13:47:34] [31merror[39m: Create Category Service:  Cannot read properties of undefined (reading 'map')
[2025-04-15 13:47:35] [31merror[39m: Create Category Service:  Cannot read properties of undefined (reading 'map')
[2025-04-15 14:11:20] [31merror[39m: Create Category Service:  Cannot read properties of undefined (reading 'map')
[2025-04-15 14:18:34] [31merror[39m: Create Category Service:  Cannot read properties of undefined (reading 'map')
[2025-04-15 14:24:54] [31merror[39m: Validation failed: [
  {
    "code": "invalid_type",
    "expected": "object",
    "received": "number",
    "path": [],
    "message": "Expected object, received number"
  }
]
[2025-04-15 14:25:37] [31merror[39m: Validation failed: [
  {
    "code": "invalid_type",
    "expected": "object",
    "received": "number",
    "path": [],
    "message": "Expected object, received number"
  }
]
[2025-04-15 14:26:50] [31merror[39m: Validation failed: [
  {
    "code": "invalid_type",
    "expected": "object",
    "received": "number",
    "path": [
      "subCategory",
      0
    ],
    "message": "Expected object, received number"
  },
  {
    "code": "invalid_type",
    "expected": "object",
    "received": "number",
    "path": [
      "subCategory",
      1
    ],
    "message": "Expected object, received number"
  }
]
[2025-04-15 14:41:23] [31merror[39m: Create Subcategory Service:  Cannot read properties of undefined (reading 'map')
[2025-04-15 14:43:11] [31merror[39m: Validation failed: [
  {
    "code": "invalid_type",
    "expected": "object",
    "received": "number",
    "path": [],
    "message": "Expected object, received number"
  }
]
[2025-04-15 14:59:24] [31merror[39m: Validation failed: [
  {
    "code": "invalid_type",
    "expected": "object",
    "received": "number",
    "path": [],
    "message": "Expected object, received number"
  }
]
[2025-04-15 15:05:30] [31merror[39m: Validation failed: [
  {
    "code": "invalid_type",
    "expected": "object",
    "received": "number",
    "path": [],
    "message": "Expected object, received number"
  }
]
[2025-04-15 15:05:47] [31merror[39m: Validation failed: [
  {
    "code": "invalid_type",
    "expected": "object",
    "received": "number",
    "path": [],
    "message": "Expected object, received number"
  }
]
[2025-04-15 15:13:02] [31merror[39m: Get Section Service:  
Invalid `prisma.section.create()` invocation in
D:\Programing\WorkFolders\restaurant-pos\backend\src\domain\section\section.repository.ts:25:25

  22 
  23 export function createSectionDB(data: Section) {
  24   const tables = data?.tables;
â†’ 25   return prisma.section.create(
The column `Table.status` does not exist in the current database.
[2025-04-15 15:13:57] [31merror[39m: Get Section Service:  
Invalid `prisma.section.create()` invocation in
D:\Programing\WorkFolders\restaurant-pos\backend\src\domain\section\section.repository.ts:25:25

  22 
  23 export function createSectionDB(data: Section) {
  24   const tables = data?.tables;
â†’ 25   return prisma.section.create(
The column `Table.status` does not exist in the current database.
[2025-04-15 15:21:40] [31merror[39m: Validation failed: [
  {
    "code": "invalid_type",
    "expected": "number",
    "received": "string",
    "path": [
      "id"
    ],
    "message": "Expected number, received string"
  }
]
[2025-04-15 15:24:01] [31merror[39m: Validation failed: [
  {
    "code": "invalid_type",
    "expected": "number",
    "received": "string",
    "path": [
      "id"
    ],
    "message": "Expected number, received string"
  }
]
[2025-04-15 15:24:03] [31merror[39m: Validation failed: [
  {
    "code": "invalid_type",
    "expected": "number",
    "received": "string",
    "path": [
      "id"
    ],
    "message": "Expected number, received string"
  }
]
[2025-04-15 15:24:06] [31merror[39m: Validation failed: [
  {
    "code": "invalid_type",
    "expected": "number",
    "received": "string",
    "path": [
      "id"
    ],
    "message": "Expected number, received string"
  }
]
[2025-04-15 16:04:10] [31merror[39m: Get Tables By ID Service:  
Invalid `prisma.table.create()` invocation in
D:\Programing\WorkFolders\restaurant-pos\backend\src\domain\table\table.repository.ts:26:23

  23 
  24 export function createTableDB(data: Table) {
  25   const { orders } = data;
â†’ 26   return prisma.table.create(
Foreign key constraint violated: `Table_sectionId_fkey (index)`
[2025-04-21 17:42:12] [31merror[39m: No Item Found
[2025-04-21 17:51:00] [31merror[39m: No Item Found
[2025-04-21 17:51:12] [31merror[39m: No Item Found
[2025-04-21 17:58:06] [31merror[39m: Create Subcategory Service:  
Invalid `prisma.subCategory.create()` invocation in
D:\Programing\WorkFolders\restaurant-pos\backend\src\domain\subcategory\subcategory.repository.ts:37:29

  34 export function createSubcategoryDB(data: SubCategory) {
  35   const items = data?.items;
  36 
â†’ 37   return prisma.subCategory.create(
Foreign key constraint violated: `SubCategory_categoryId_fkey (index)`
[2025-04-21 18:00:00] [31merror[39m: Create Subcategory Service:  
Invalid `prisma.subCategory.create()` invocation in
D:\Programing\WorkFolders\restaurant-pos\backend\src\domain\subcategory\subcategory.repository.ts:37:29

  34 export function createSubcategoryDB(data: SubCategory) {
  35   const items = data?.items;
  36 
â†’ 37   return prisma.subCategory.create(
Foreign key constraint violated: `SubCategory_categoryId_fkey (index)`
[2025-04-22 14:59:56] [31merror[39m: Create Subcategory Service:  
Invalid `prisma.subCategory.create()` invocation in
D:\Programing\WorkFolders\restaurant-pos\backend\src\domain\subcategory\subcategory.repository.ts:35:29

  32 }
  33 
  34 export function createSubcategoryDB(data: SubCategory) {
â†’ 35   return prisma.subCategory.create(
Null constraint violation on the fields: (`categoryId`)
[2025-04-22 15:02:29] [31merror[39m: Menu item Service:  
Invalid `prisma.menuItem.update()` invocation in
D:\Programing\WorkFolders\restaurant-pos\backend\src\domain\item\item.repository.ts:31:26

  28 }
  29 
  30 export async function updateItemDB(id: number, data: MenuItem) {
â†’ 31   return prisma.menuItem.update(
Foreign key constraint violated: `MenuItem_subCategoryId_fkey (index)`
[2025-04-22 17:50:02] [31merror[39m: No Item Found
[2025-04-26 14:23:53] [31merror[39m: Validation failed: [
  {
    "code": "invalid_type",
    "expected": "string",
    "received": "undefined",
    "path": [
      "password"
    ],
    "message": "Required"
  }
]
[2025-04-26 14:23:53] [31merror[39m: Auth login Service:  [
  {
    "code": "invalid_type",
    "expected": "string",
    "received": "undefined",
    "path": [
      "password"
    ],
    "message": "Required"
  }
]
[2025-04-26 16:07:27] [31merror[39m: Validation failed: [
  {
    "code": "invalid_type",
    "expected": "string",
    "received": "undefined",
    "path": [
      "password"
    ],
    "message": "Required"
  }
]
[2025-04-26 16:07:53] [31merror[39m: Validation failed: [
  {
    "code": "invalid_type",
    "expected": "string",
    "received": "undefined",
    "path": [
      "username"
    ],
    "message": "Required"
  },
  {
    "code": "invalid_type",
    "expected": "string",
    "received": "undefined",
    "path": [
      "password"
    ],
    "message": "Required"
  }
]
[2025-04-26 16:08:09] [31merror[39m: Validation failed: [
  {
    "code": "invalid_type",
    "expected": "string",
    "received": "undefined",
    "path": [
      "username"
    ],
    "message": "Required"
  }
]
[2025-04-26 17:28:01] [31merror[39m: Error uploading image:  Cannot read properties of undefined (reading 'toLowerCase')
[2025-04-26 17:29:11] [31merror[39m: Error uploading image:  Cannot read properties of undefined (reading 'toLowerCase')
[2025-04-26 17:30:10] [31merror[39m: Error uploading image:  No value provided for input HTTP label: Bucket.
[2025-04-26 17:30:40] [31merror[39m: Error uploading image:  No value provided for input HTTP label: Bucket.
[2025-04-26 17:31:36] [31merror[39m: Error uploading image:  Could not load credentials from any providers
[2025-04-26 17:34:31] [31merror[39m: Error uploading image:  Could not load credentials from any providers
[2025-04-26 17:36:38] [31merror[39m: Error uploading image:  Could not load credentials from any providers
[2025-04-28 15:42:40] [31merror[39m: Validation failed: [
  {
    "code": "invalid_type",
    "expected": "string",
    "received": "undefined",
    "path": [
      "company"
    ],
    "message": "Required"
  }
]
[2025-04-28 15:43:30] [31merror[39m: Menu item Service:  
Invalid `prisma.menuItem.update()` invocation in
D:\Programing\WorkFolders\restaurant-pos\backend\src\domain\item\item.repository.ts:77:26

  74   data: MenuItem,
  75   include?: Prisma.MenuItemInclude
  76 ) {
â†’ 77   return prisma.menuItem.update(
Foreign key constraint violated: `MenuItem_subCategoryId_fkey (index)`
[2025-04-28 15:43:45] [31merror[39m: Menu item Service:  
Invalid `prisma.menuItem.update()` invocation in
D:\Programing\WorkFolders\restaurant-pos\backend\src\domain\item\item.repository.ts:77:26

  74   data: MenuItem,
  75   include?: Prisma.MenuItemInclude
  76 ) {
â†’ 77   return prisma.menuItem.update(
Foreign key constraint violated: `MenuItem_subCategoryId_fkey (index)`
[2025-04-28 15:44:12] [31merror[39m: Menu item Service:  
Invalid `prisma.menuItem.update()` invocation in
D:\Programing\WorkFolders\restaurant-pos\backend\src\domain\item\item.repository.ts:77:26

  74   data: MenuItem,
  75   include?: Prisma.MenuItemInclude
  76 ) {
â†’ 77   return prisma.menuItem.update(
Foreign key constraint violated: `MenuItem_subCategoryId_fkey (index)`
[2025-04-28 15:44:40] [31merror[39m: Menu item Service:  
Invalid `prisma.menuItem.update()` invocation in
D:\Programing\WorkFolders\restaurant-pos\backend\src\domain\item\item.repository.ts:77:26

  74   data: MenuItem,
  75   include?: Prisma.MenuItemInclude
  76 ) {
â†’ 77   return prisma.menuItem.update(
Foreign key constraint violated: `MenuItem_subCategoryId_fkey (index)`
[2025-04-28 15:45:21] [31merror[39m: Validation failed: [
  {
    "code": "invalid_type",
    "expected": "number",
    "received": "undefined",
    "path": [
      "subCategoryId"
    ],
    "message": "Required"
  }
]
[2025-04-28 15:45:34] [31merror[39m: Menu item Service:  
Invalid `prisma.menuItem.update()` invocation in
D:\Programing\WorkFolders\restaurant-pos\backend\src\domain\item\item.repository.ts:77:26

  74   data: MenuItem,
  75   include?: Prisma.MenuItemInclude
  76 ) {
â†’ 77   return prisma.menuItem.update(
Foreign key constraint violated: `MenuItem_subCategoryId_fkey (index)`
[2025-04-28 15:46:03] [31merror[39m: Menu item Service:  
Invalid `prisma.menuItem.update()` invocation in
D:\Programing\WorkFolders\restaurant-pos\backend\src\domain\item\item.repository.ts:77:26

  74   data: MenuItem,
  75   include?: Prisma.MenuItemInclude
  76 ) {
â†’ 77   return prisma.menuItem.update(
Foreign key constraint violated: `MenuItem_subCategoryId_fkey (index)`
